azurelinuxagent/ga/update.py

```diff
@@ -752,6 +752,12 @@ def log_if_agent_versioning_feature_disabled():
             log_if_op_disabled("OS.EnableFirewall", conf.enable_firewall())
             log_if_op_disabled("Extensions.Enabled", conf.get_extensions_enabled())
             log_if_op_disabled("AutoUpdate.Enabled", conf.get_autoupdate_enabled())
+            log_if_op_disabled("AutoUpdate.UpdateToLatestVersion", conf.get_auto_update_to_latest_version())
+
+            if not conf.get_autoupdate_enabled():
+                msg = "AutoUpdate.Enabled property is **Deprecated** but it was used and set to False. Please switch to using AutoUpdate.UpdateToLatestVersion instead."
```

Comment:
Should we include wiki link here too? This message makes it sound like AutoUpdate.Enabled and AutoUpdate.UpdateToLatestVersion do the same thing